srcs/builtins/cd_builtin.c

```diff
@@ -62,3 +64,14 @@ int	cd_builtin(t_shell *shell, char **args)
 	free(path);
 	return (0);
 }
+
+// test in your terminal how command "cd $PWD {sone existing dir name}" works
```

Comment:
You're right, in bash I'm able to switch dir with cd $PWD/includes while our cd can't handle that...
However, it's not enough to add this static function. It needs to be invoked from the cd_builtin when there are 2 args and one of them is $PWD. I'll work on that :)